airbyte-integrations/connectors/source-bing-ads/source_bing_ads/manifest.yaml

```diff
@@ -144,8 +143,120 @@ definitions:
               - LastModifiedTime
             value: '{{ format_datetime(record.LastModifiedTime, "%Y-%m-%dT%H:%M:%S.%f" ) }}'
 
+  campaigns_stream:
+    type: DeclarativeStream
+    name: campaigns
+    primary_key: Id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/schemas/campaigns"
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://campaign.api.bingads.microsoft.com
+        path: /CampaignManagement/v13/Campaigns/QueryByAccountId
+        http_method: POST
+        request_headers:
+          Content-Type: application/json
+          CustomerAccountId: "{{ stream_partition['account_id'][0] }}"
+          CustomerId: "{{ stream_slice.extra_fields['ParentCustomerId'][0] }}"
+          DeveloperToken: "{{ config['developer_token'] }}"
+          Authorization: "Bearer {{ config['_token']['access_token'] }}"
+        request_body_json:
+          AccountId: "{{ stream_partition['account_id'][0] }}"
+          CampaignType: "Audience,DynamicSearchAds,Search,Shopping,PerformanceMax"
+          ReturnAdditionalFields: "AdScheduleUseSearcherTimeZone,BidStrategyId,CpvCpmBiddingScheme,DynamicDescriptionSetting,DynamicFeedSetting,MaxConversionValueBiddingScheme,MultimediaAdsBidAdjustment,TargetImpressionShareBiddingScheme,TargetSetting,VerifiedTrackingSetting"
+        authenticator: "#/definitions/authenticator"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["Campaigns"]
+          schema_normalization: Default
+      partition_router:
+        type: GroupingPartitionRouter
+        group_size: 1 # If you want to query one account at a time
+        deduplicate: true
+        underlying_partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: Id
+              partition_field: account_id
+              stream:
+                $ref: "#/definitions/accounts_stream"
+              extra_fields:
+                - - ParentCustomerId
+    transformations:
+      - type: AddFields
+        fields:
+          - type: AddedFieldDefinition
+            path:
+              - Id
+            value: "{{ record.Id|int }}"
```

Comment:
I initially thought why we need this transformation if we have set default schema normalization, but understood that schema defines type as numbers which is weird, but schema was added before, so  I think it's acceptable for now.